tests/graphql_definitions/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,52 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`fields.graphql 1`] = `
+query MyFirstQuery {
+  hello
+}
+
+mutation
+MyFirstMutation {
+  world
+}
+
+subscription, ThisIsASub, {
+  excellent
+}
+
+, query, ThisIsASub, {
```

Comment:
Yeah this is a greatest hit as well: https://github.com/facebook/graphql/pull/226